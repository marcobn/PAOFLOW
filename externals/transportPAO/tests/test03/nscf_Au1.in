Infinite Gold Chain
1 atom/cell, Lattice Parameter = 2.5 Ang
   &CONTROL
      calculation = 'nscf',
      restart_mode='from_scratch',
      verbosity='default',
      prefix='gold1',
      nstep=100,
      tstress = .false.,
      tprnfor = .true.,
      pseudo_dir = '../pseudo',
      outdir = './SCRATCH/'
      wf_collect = .TRUE.
   /
   &SYSTEM
      ibrav=8, 
      celldm(1)=20.0, 
      celldm(2)=1.0,
      celldm(3)=0.236216
      nat=1, 
      ntyp=1,
      ecutwfc=50.0,
      ecutrho=200.0,
      occupations='smearing',
      smearing='gaussian',
      degauss=0.01,
      nbnd=10
      noinv = .TRUE.
      nosym = .TRUE.
   /
   &ELECTRONS
      electron_maxstep = 100,
      mixing_mode = 'plain',
      mixing_beta = 0.3,
      conv_thr =  1.0d-8,
   /
ATOMIC_SPECIES
  Au   196.9665   Au11pw91.mt.UPF
ATOMIC_POSITIONS {angstrom}
Au   0.0  0.0 0.0
K_POINTS {automatic}
1 1 8  0 0 0

