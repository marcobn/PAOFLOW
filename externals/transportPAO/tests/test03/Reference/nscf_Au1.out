
     Program PWSCF v.5.0.2 (svn rev. 9392) starts on 17Apr2014 at 20:19: 9 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Reading input from /home/ferretti/want/tests/test16/nscf_Au1.in

     Atomic positions and unit cell read from directory:
     ./SCRATCH/gold1.save/
 

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        1061    1061    316                15056    15056    2465
     Max        1063    1063    317                15057    15057    2466
     Sum        6369    6369   1901                90341    90341   14791
 


     bravais-lattice index     =            8
     lattice parameter (alat)  =      20.0000  a.u.
     unit-cell volume          =    1889.7280 (a.u.)^3
     number of atoms/cell      =            1
     number of atomic types    =            1
     number of electrons       =        11.00
     number of Kohn-Sham states=           10
     kinetic-energy cutoff     =      50.0000  Ry
     charge density cutoff     =     200.0000  Ry
     Exchange-correlation      =  SLA  PW   GGX  GGC ( 1 4 2 2 0)
     EXX-fraction              =        0.00

     celldm(1)=  20.000000  celldm(2)=   1.000000  celldm(3)=   0.236216
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   0.236216 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  4.233413 )  


     PseudoPot. # 1 for Au read from file:
     ../pseudo/Au11pw91.mt.UPF
     MD5 check sum: e270e13d0e45fa10676aee720c7982d9
     Pseudo is Norm-conserving + core correction, Zval = 11.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  331 points,  2 beta functions with: 
                l(1) =   1
                l(2) =   2

     atomic species   valence    mass     pseudopotential
        Au            11.00   196.96650     Au( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Au  tau(   1) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=     8  gaussian smearing, width (Ry)=  0.0100
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.2500000
        k(    2) = (   0.0000000   0.0000000   0.5291767), wk =   0.2500000
        k(    3) = (   0.0000000   0.0000000   1.0583534), wk =   0.2500000
        k(    4) = (   0.0000000   0.0000000   1.5875301), wk =   0.2500000
        k(    5) = (   0.0000000   0.0000000  -2.1167067), wk =   0.2500000
        k(    6) = (   0.0000000   0.0000000  -1.5875301), wk =   0.2500000
        k(    7) = (   0.0000000   0.0000000  -1.0583534), wk =   0.2500000
        k(    8) = (   0.0000000   0.0000000  -0.5291767), wk =   0.2500000

     Dense  grid:    90341 G-vectors     FFT dimensions: (  96,  96,  24)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         0.29 Mb     (   1873,   10)
        NL pseudopotentials             0.23 Mb     (   1873,    8)
        Each V/rho on FFT grid          0.56 Mb     (  36864)
        Each G-vector array             0.11 Mb     (  15057)
        G-vector shells                 0.03 Mb     (   3914)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         1.14 Mb     (   1873,   40)
        Each subspace H/S matrix        0.02 Mb     (  40,  40)
        Each <psi_i|beta_j> matrix      0.00 Mb     (      8,   10)

     Check: negative/imaginary core charge=   -0.000021    0.000000

     The potential is recalculated from file :
     ./SCRATCH/gold1.save/charge-density.dat

     Starting wfc are    9 randomized atomic wfcs

     total cpu time spent up to now is        0.3 secs

     per-process dynamical memory:    12.4 Mb

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  9.09E-11,  avg # of iterations = 44.8

     total cpu time spent up to now is        5.9 secs

     End of band structure calculation

          k = 0.0000 0.0000 0.0000     band energies (ev):

   -10.8645  -6.8159  -6.8158  -6.4586  -5.2389  -5.2389  -1.1776  -1.1776
     1.1448   2.1145

          k = 0.0000 0.0000 0.5292     band energies (ev):

   -10.0621  -6.7221  -6.7220  -6.2262  -5.9487  -5.9487  -0.4785  -0.4785
     1.5130   2.4831

          k = 0.0000 0.0000 1.0584     band energies (ev):

    -7.8773  -7.2089  -7.2089  -6.4889  -6.4887  -5.4980   1.0401   1.0401
     2.6188   3.5889

          k = 0.0000 0.0000 1.5875     band energies (ev):

    -8.1805  -8.1805  -6.2453  -6.2451  -5.8395  -3.3672   3.0327   3.0327
     4.4734   5.4316

          k = 0.0000 0.0000-2.1167     band energies (ev):

    -8.5343  -8.5343  -6.1411  -6.1409  -5.3545  -0.2765   2.0667   4.7527
     4.7527   6.9997

          k = 0.0000 0.0000-1.5875     band energies (ev):

    -8.1805  -8.1805  -6.2453  -6.2451  -5.8395  -3.3672   3.0327   3.0327
     4.4734   5.4316

          k = 0.0000 0.0000-1.0584     band energies (ev):

    -7.8773  -7.2089  -7.2089  -6.4889  -6.4887  -5.4980   1.0401   1.0401
     2.6188   3.5889

          k = 0.0000 0.0000-0.5292     band energies (ev):

   -10.0621  -6.7221  -6.7220  -6.2262  -5.9487  -5.9487  -0.4785  -0.4785
     1.5130   2.4831

     the Fermi energy is    -5.2271 ev

     Writing output data file gold1.save
 
     init_run     :      0.21s CPU      0.20s WALL (       1 calls)
     electrons    :      5.52s CPU      5.54s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.01s CPU      0.00s WALL (       1 calls)
     potinit      :      0.07s CPU      0.08s WALL (       1 calls)

     Called by electrons:
     c_bands      :      5.52s CPU      5.54s WALL (       1 calls)
     v_of_rho     :      0.05s CPU      0.06s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.01s WALL (       8 calls)
     cegterg      :      5.23s CPU      5.25s WALL (      21 calls)

     Called by *egterg:
     h_psi        :      4.59s CPU      4.61s WALL (     387 calls)
     g_psi        :      0.12s CPU      0.06s WALL (     358 calls)
     cdiaghg      :      0.20s CPU      0.19s WALL (     366 calls)

     Called by h_psi:
     add_vuspsi   :      0.07s CPU      0.04s WALL (     387 calls)

     General routines
     calbec       :      0.08s CPU      0.07s WALL (     387 calls)
     fft          :      0.02s CPU      0.03s WALL (      11 calls)
     fftw         :      3.97s CPU      4.00s WALL (    2942 calls)
     davcio       :      0.00s CPU      0.01s WALL (      16 calls)
 
     Parallel routines
     fft_scatter  :      1.22s CPU      1.12s WALL (    2953 calls)
 
     PWSCF        :     6.01s CPU         6.17s WALL

 
   This run was terminated on:  20:19:15  17Apr2014            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
